controllers/articles.js

```diff
@@ -756,6 +763,14 @@ const articles = {
       link
     });
     open(`mailto:?subject=${title}&body=${link}`);
+  },
+
+  getTags: async (req, res) => {
```

Comment:
@fridolinho Good work Fridz

For the tags, you made one endpoint for getting all tags at the same time. I think you should add another endpoint in case user want to find all tags based to a certain keyword s/he can provide `findAll({where: x})` because this can be used for searching tags match provided keyword.

Reply:
Good, that will help us in the future.